Tests/TuistKitTests/ProjectEditor/ProjectEditorMapperTests.swift

```diff
@@ -0,0 +1,114 @@
+import Basic
+import Foundation
+import TuistCore
+import TuistSupport
+import XCTest
+
+@testable import TuistKit
+@testable import TuistSupportTesting
+
+final class ProjectEditorMapperTests: TuistUnitTestCase {
+    var subject: ProjectEditorMapper!
+
+    override func setUp() {
+        super.setUp()
+        subject = ProjectEditorMapper()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        subject = nil
+    }
+
+    func test_edit_when_there_are_helpers() throws {
+        // Given
+        let sourceRootPath = try temporaryPath()
+        let manifestPaths = [sourceRootPath].map { $0.appending(component: "Project.swift") }
+        let helperPaths = [sourceRootPath].map { $0.appending(component: "Project+Template.swift") }
+        let projectDescriptionPath = sourceRootPath.appending(component: "ProjectDescription.framework")
+
+        // When
+        let (project, graph) = subject.map(sourceRootPath: sourceRootPath,
+                                           manifests: manifestPaths,
+                                           helpers: helperPaths,
+                                           projectDescriptionPath: projectDescriptionPath)
+
+        // Then
+        let manifestsTarget = project.targets.first
+        let helpersTarget = project.targets.last
+
+        XCTAssertEqual(project.name, "Manifests")
+        XCTAssertEqual(project.filesGroup, .group(name: "Manifests"))
+
+        XCTAssertEqual(manifestsTarget?.name, "Manifests")
```

Comment:
nit: if the `manifestsTarget` is Equatable, you can do one single equal assertion against the expected Manifest. If you want the Assert to give you a more granular failure, there is a very cool gist from objc.io that uses `Diffing` and extends the system `XCTAssertEqual` so that it tells where two objects are different

Reply:
> If you want the Assert to give you a more granular failure, there is a very cool gist from objc.io that uses Diffing and extends the system XCTAssertEqual so that it tells where two objects are different

Do you have a link to this one?

Reply:
Ups, I was wrong, it's not objcio :) It's this one https://github.com/krzysztofzablocki/Difference